CHANGELOG.md

```diff
@@ -117,13 +119,12 @@
  - BayesFilters is now fully relocatable.
 
 
-## Version 0.6.2.0
-### Updates
+## 🔖 Version 0.6.2.0
 ##### `Filtering classes`
  - Added ResamplingWithPrior class.
  - Added HistoryBuffer and EstimatesExtraction classes.
 
-##### `Filtering Features`
+##### `Filtering features`
```

Comment:
Since in this commit we are cleaning a bit the Changelog, can we also change `Filtering Functions` to `Filtering functions` and the same for `Algorithms` and `Utilities` in order to enforce this convention?

Reply:
What do you mean? You mean change the capital letter?

Reply:
Yes, the capital letter of the second word.

Reply:
Done!